department-of-veterans-affairs/abd-vro
# MCP-2795: Refactor receipt date extraction

## What was the problem?

To minimize changes, only service-location receipt date was changed, in https://github.com/department-of-veterans-affairs/abd-vro/pull/1479

Associated tickets or Slack threads:

- [MCP-2795](https://amida.atlassian.net/browse/MCP-2795)

## How does this fix it?[^1]

We can reformat receiptDate per annotation document and remove some downstream code in the assessment services. 

## How to test this PR
- Step 1 Run automated claim, example, (376)
- Step 2 Download PDF and verify that recieptDate is formated (m/d/yyyy)
- [^secrel]


[^1]: [Pull-Requests guidelines](https://github.com/department-of-veterans-affairs/abd-vro/wiki/Pull-Requests). If PR is significant, update [Current Software State](https://github.com/department-of-veterans-affairs/abd-vro/wiki/Current-Software-State) wiki page.
[^secrel]: To check if a PR will succeed in the SecRel workflow, [test PRs in the SecRel pipeline](https://github.com/department-of-veterans-affairs/abd-vro-internal/wiki/Secure-Release-process#to-test-prs-in-the-secrel-pipeline).
